help/access/databases.md

```diff
@@ -305,194 +313,241 @@ supported. You can add an input/output of type `list<string>`.
+1. Select how to store data:
+    * Save the data as a static snapshot.
+    * Store the data as a generation script by toggling the **Data sync** control. The query re-executes each time the project is opened. To learn more about dynamic data updates in projects, see [Dynamic data](../datagrok/project.md/#dynamic-data).
+1. Click **OK** to upload the project.
+
+![Dynamic dashboards](dynamic-dashboards.gif)
+
+## Sharing query results
+
+You have two options to share query results in Datagrok:
+
+* Share its URL
+* Share it as a project.
+
+Each query output has a unique URL, which has all the information needed to recreate the query, including its parameters. This can be especially useful for parameterized queries, where you can embed the query output on external websites with specific parameters applied. After you have executed a query, copy the URL from the address bar and share it with others. When this URL is accessed, a query gets re-executed, so the query output always displays the latest data. When the users open the report, they can still change the query parameters in the **Toolbox** on the left.
+
+:::note
+
+To access the query results from the link provided, users must have the necessary permissions to execute this query.
+
+:::
+
+If you want to persist a specific layout, create a dynamic dashboard, or assign access permissions to specific groups or users, you can share query results as a project. First, you need to [upload the project](#creating-views-for-query-results). Then, locate the desired project in **Data** > **Projects** and right-click it to share. To learn more about access privileges, see [Access control](#access-control).
+
+## Access control
+
+In Datagrok, certain objects called _entities_ have a [set of operations](../datagrok/objects.md) that can be applied to them. These entities are connections, queries, tables, and table columns, all of which can be shared, assigned permissions, annotated, and more.
+
+When you create an entity such as a connection or a query, it's private and visible to you only. To make it accessible to others, you can share it. 
+
+To share:
+
+1. Right-click the connection or a query you want to share and select **Share...**.
+2. In the **Share** dialog, enter a user or a group that needs access and set corresponding permissions. These privileges can be given to individuals or to [groups](../govern/group.md) (which can be defined via dynamic filters). For more information on the access privilege model, see [Privileges](../govern/security.md#privileges).
+3. Optional. Add a description in the provided text field. If you don't want to notify the recipients, clear the **Send notification** checkbox.
+   :::note
+
+   To notify via email, enter the user's email in the identity/email field. The email contains a link to the shared item and entered description. When you enter a user or a group name, they are  notified via the Datagrok interface.
```

Comment:
Nope, this is a much cooler feature :)

To share an object with a user that does not have a Datagrok account yet, enter the user's email. The user will get an email about it, and when they sign up the object will be available to them.